### ISSUE TITLE
Verify by mention - allowing bots to introduce new members to a server 

### ISSUE THREAD

**Issue by Velocitractor00** [PokeNavBot/Issue-Tracker#600]
**Is your feature request related to a problem? Please describe.**
specifically around new members joining a server, and not so much verifying existing users 

I would like to integrate the verify command into my existing welcome process, which is a ticket system created by atlas, just now (on one of the servers at least) when the ticket is created we do a silphcard and trainer code lookup - automatically 

if verify could be integrated into that (primary request) - this would allow us to have the new channel that is created as the ticket, run $verify followed by tagging the user, triggering the process for them to verify and the steps 

(secondary request) would be to include the silphcard being set as part of the verification process - we use these primarily to reflect the correct nickname onto each user (as silphcard matches ign) 

**Describe the solution you'd like**
verify by mention - even if its only by bots 
(or more long term - pokenav to have its own ticket system where it asks the user to type verify to start the process inside a unique channel) 

**Describe alternatives you've considered**
just now its more of a manual review process, this would allow us to automate it, and even if I have to go back and historically change the nicknames to match the IGN it would still provide a record of these for people as they join the server 

**Additional context**
I feel that this ties well into daadybarns ticket about nickname support - but I don't know of a decent way to integrate that, other than asking the user as part of the verify process "what is your ign" and then using that info to set their nickname - if thats possible, that would be astronomically helpful on all servers imo.

**Comment by daadyrbarn** [comment]
Your secondary point could technically be achieved by having PokeNav automatically assume that IGN == SilphCard and set it as part of `$set ign`. 
Or maybe that's what you meant in the first place :D

**Comment by Velocitractor00** [comment]
yeah, thats what I was getting at, it sort of lines up with your request

**Comment by dandesousa** [comment]
I created #602 to track the Silph card requirement, we can elaborate in there if needed. I would like to do some sort of silphlard == ign assumption. The way I think it should probably work is that it becomes a part of silph sync, so that if your in game name is not set it pulls it from the Silph card and when you set your ign for the first time it will look for a Silph card that matches if Silph sync is set and tie it to that Silph card as well.

**Comment by Velocitractor00** [comment]
That would be tight!

**Comment by dandesousa** [comment]
Verify by mention is definitely something that I think could work, but I think I would make it a new command with different permissions. I don't think we want regular users doing this for others.

I'm thinking limited to either admins (or manager server perms) or bots, one concurrent command running per channel its invoked in. I think this is something we could do until (or instead of) the ticketing system or verification in PokeNav, depending on how much desire there is for verification via separate channels in the community.

How does that sound
